rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/openstack/OpenstackBakeHandlerSpec.groovy

```diff
@@ -0,0 +1,679 @@
+/*
+ * Copyright 2015 Google, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.rosco.providers.openstack
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.netflix.spinnaker.rosco.api.Bake
+import com.netflix.spinnaker.rosco.api.BakeRequest
+import com.netflix.spinnaker.rosco.config.RoscoConfiguration
+import com.netflix.spinnaker.rosco.providers.openstack.config.RoscoOpenstackConfiguration
+import com.netflix.spinnaker.rosco.providers.util.ImageNameFactory
+import com.netflix.spinnaker.rosco.providers.util.PackerCommandFactory
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Subject
+import spock.lang.Unroll
+
+class OpenstackBakeHandlerSpec extends Specification {
+
+  private static final String PACKAGES_NAME = "kato nflx-djangobase-enhanced_0.1-h12.170cdbd_all mongodb"
+  private static final String REGION = "TTEOSCORE1"
+  private static final String SOURCE_UBUNTU_IMAGE_NAME = "ubuntu-latest"
+  private static final String SOURCE_TRUSTY_IMAGE_NAME = "ubuntu-trusty"
+  private static final String SOURCE_OPENSTACK_IMAGE_NAME = "rhel7-latest"
+  private static final String DEBIAN_REPOSITORY = "http://some-debian-repository"
+  private static final String YUM_REPOSITORY = "http://some-yum-repository"
+  private static final String OPENSTACK_ENDPOINT = "http://my.openstack.com:5000/v3"
+
+  @Shared
+  String configDir = "/some/path"
+
+  @Shared
+  RoscoOpenstackConfiguration.OpenstackBakeryDefaults openstackBakeryDefaults
+
+  @Shared
+  RoscoConfiguration roscoConfiguration
+
+  void setupSpec() {
+    def openstackBakeryDefaultsJson = [
```

Comment:
Please make the indentation here and everywhere match the neighbors: https://github.com/spinnaker/rosco/blob/master/rosco-core/src/test/groovy/com/netflix/spinnaker/rosco/providers/google/GCEBakeHandlerSpec.groovy#L55